src/lib.rs

```diff
@@ -481,7 +485,19 @@ mod test {
         assert_eq!(decoded_result.unwrap(), hash1);
     }  
 
-    #[cfg(feature = "bench")] 
+    #[test]
+    fn base64_error_on_empty() {
+        let decoded_result = ImageHash::from_base64("");
+        assert!(decoded_result.is_err());
+        /* FIXME: Figure out how to make the below run
+        match decoded_result {
+            Err(FromBase64Error::InvalidBase64Length) => assert!(true),
```

Comment:
Oh yeah, you need to duplicate the import of `FromBase64Error` in the sub-module.

In fact, you can just do `use serialize::base64::*;` for both the `test` submodule and the top level, since we're using a majority of the definitions anyway.